jboss/container/wildfly/galleon/build-feature-pack/configure.sh

```diff
@@ -34,7 +34,9 @@ if [ -f "${ZIPPED_REPO}" ]; then
   unzip ${ZIPPED_REPO} -d /tmp
   repoDir=$(find /tmp -type d -iname "*-image-builder-maven-repository")
   mv $repoDir/maven-repository "$TMP_GALLEON_LOCAL_MAVEN_REPO"
```

Comment:
I read that as move everything in `$repoDir` other than `maven-repository` to `$JBOSS_CONTAINER_WILDFLY_S2I_GALLEON_DIR/maven-repo-misc`.

Is this only the licenses or other other things moved?

Reply:
We could have some more, eg: the patches applied, JBossEULA.txt file.

Reply:
I'm fine with the PR but the title and commit message should reflect that this deals with any extra stuff (not only licenses)